WalletWasabi.Gui/Controls/QrCode.cs

```diff
@@ -1,20 +1,43 @@
 using Avalonia;
 using Avalonia.Controls;
 using Avalonia.Media;
+using Avalonia.Media.Imaging;
 using Avalonia.Metadata;
+using Avalonia.Platform;
+using ReactiveUI;
 using System;
+using System.Reactive.Linq;
 
 namespace WalletWasabi.Gui.Controls
 {
 	public class QrCode : Control
 	{
+		private readonly int _matrixPadding;
```

Comment:
```suggestion
		private readonly int MatrixPadding;
```
@nopara73 Should we use `readonly`?

Reply:
It's not going to be modified at runtime. It should be a constant i think

Reply:
I made it into a constant now